config/settings.py

```diff
@@ -4,3 +4,4 @@
 ROLLBAR_ACCESS_TOKEN = (
     os.getenv('ROLLBAR_ACCESS_TOKEN', None) if IS_PRODUCTION and not IS_TEST else None
 )
+ENABLE_ROLLBAR = ROLLBAR_ACCESS_TOKEN is not None
```

Comment:
@tahitihat this feels a bit like a snake eating it's head - but I guess we do need both. You can't configure rollbar without the token, and you we need a way to know if rollbar is enabled or not. (unless there's a way to interrogate it)